examples/topics/batched_example/GSSHA_Workflow_Batched_Example1.ipynb

```diff
@@ -30,9 +30,17 @@
     "PARAM_JSON_INIT='{\"rain_intensity\":42, \"rain_duration\":50}' jupyter notebook GSSHA_Workflow_Batched_Example1.ipynb\n",
     "```\n",
     "\n",
+    "You can also generate a report HTML file using ``nbconvert`` as follows:\n",
+    "\n",
+    "```bash\n",
+    "PARAM_JSON_INIT='{\"rain_intensity\":25, \"rain_duration\":60}' jupyter nbconvert --execute GSSHA_Workflow_Batched_Example1.ipynb --output report1.html\n",
```

Comment:
Presumably `--output report1.html` is optional?

Reply:
Yes. It would output the notebook name with the ``.html`` extension otherwise.

It seemed worth showing this flexibility but I can also mention that is it optional if you wish.

Reply:
I think omitting it here is correct, because then the example shows the right way to do the simple case.  And then the Lancet example can show how to encode parameter names into the filenames as a record, if appropriate.